buildlib/azure-pipelines-perf.yml

```diff
@@ -130,3 +165,22 @@ stages:
               set -x
               rm -rf $(WorkDir)
             displayName: Cleanup WorkDir
+
+      - job: RemoveWarning
```

Comment:
shall we set condition to always run this step?

Reply:
Already set at the parent stage level